Assets/GLTF/Scripts/GLTFComponent.cs

```diff
@@ -1,14 +1,25 @@
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using UnityEngine;
 using System.Threading;
+using System.Linq;
 using UnityEngine.Networking;
 
 namespace GLTF {
 
 	class GLTFComponent : MonoBehaviour
 	{
+		[Serializable]
```

Comment:
Why is this Serializable?

Reply:
If I remember correctly, the struct needed to be serializable in order apply reflection in the consequent operations, this may not be needed at all. Not remember for sure why I did.